containers/no_data_vec.rs

```diff
@@ -1,13 +1,9 @@
 use std::marker::PhantomData;
 use std::ops::Index;
-use {
-    crate::{
-        nondet::{
-            Nondet, nondet, nondet_pointer
-        },
-    }
-};
-use crate::{CVT_assume};
+use nondet::{
+        Nondet, nondet, nondet_pointer
+    };
+use cvt_core::CVT_assume;
 
 ////////////////////////////////////////////////////////////////////////
 // Adapted from Kani.
```

Comment:
remove this comment. rewrite the implementation from scratch to avoid dependency on third party

Reply:
the `no_data_vec` implementation? 
If yes, I think that should be a separate PR?

Reply:
also how is it third party dependent? This file only depends on `cvt`, `nondet` and `std` crates